src/containers/Profile/Profile.js

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import EventContainer from '../EventContainer/EventContainer';
+import { setUser } from '../../actions';
+import { mockUser } from '../../data/mockData';
 
 export const Profile = () => {
+  const [contentEditable, toggleEditable] = useState(false);
+  const buttonText = contentEditable ? 'Save' : 'Edit Profile';
 
   return (
-    <div></div>
-  )
-}
+    <div className="Profile">
+      <div className="user-intro-section">
+        <img
+          src={mockUser.image_url}
+          alt="user headshot"
+          className="user-photo"
+        />
+        <h1 className="user-name">{mockUser.name.toUpperCase()}</h1>
+      </div>
+      <div className="user-bio-section">
+        <p className="user-about-me">About Me</p>
+        <button
+          onClick={() => toggleEditable(!contentEditable)}
+          className="edit-profile-button"
+        >
+          {buttonText}
+        </button>
+        <p className="user-bio" contentEditable={contentEditable}>
```

Comment:
nice use of hooks here to set contentEditable! Looks so clean. Do we want to add content editable to the user name as well so the use can edit that as well?